addons/cis-hardening/enable

```diff
@@ -419,15 +420,18 @@ def main(kubebench_version: str, install_kubebench):
     ApiServerToKubeletCertificate()
     EnableRBAC()
     CopyExtraConfigFiles()
-    if install_kubebench:
+
+    should_install_kubebench = not (install_kubebench.lower() in ["", "false"] or skip_kubebench_installation)
```

Comment:
It's my turn then to request a one-liner to be split. I can understand why it works, but operator precedence rules make me think twice when looking at it. How about

```suggestion
    should_install_kubebench = install_kubebench.lower() not in ["", "false"] and not skip_kubebench_installation
```

Reply:
Sounds good!